.changeset/poor-lemons-cheer.md

```diff
@@ -0,0 +1,6 @@
+---
+'@microsoft/atlas-site': minor
+'@microsoft/atlas-css': minor
+---
+
+Add support for a vertical gradient. Also adds a new red and purple color to create a red-purple gradient for the new Ask Learn icon
```

Comment:
Can we remove Ask Learn from the changeset? I think mentioning a new red and purple gradient and vertical gradient should be sufficient.

Reply:
updated